gallery/src/components/NxGlobalSidebar/NxGlobalSidebarPage.tsx

```diff
@@ -163,6 +161,89 @@ export default function NxGlobalSidebarPage() {
             </NxTable.Body>
           </NxTable>
         </NxTile.Subsection>
+        <NxTile.Subsection>
+          <NxTile.SubsectionHeader>
+            <h3 className="nx-h3"><NxCode>NxGlobalSidebarFooter</NxCode> Props</h3>
+          </NxTile.SubsectionHeader>
+          <NxTable>
+            <NxTable.Head>
+              <NxTable.Row>
+                <NxTable.Cell>Prop</NxTable.Cell>
+                <NxTable.Cell>Type</NxTable.Cell>
+                <NxTable.Cell>Required</NxTable.Cell>
+                <NxTable.Cell>Details</NxTable.Cell>
+              </NxTable.Row>
+            </NxTable.Head>
+            <NxTable.Body>
+              <NxTable.Row>
+                <NxTable.Cell>showSupport</NxTable.Cell>
+                <NxTable.Cell>Boolean</NxTable.Cell>
+                <NxTable.Cell>Yes</NxTable.Cell>
+                <NxTable.Cell>
+                  If true the support link is shown. At minimum <NxCode>supportLink</NxCode> below should be
+                  provided.
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>supportIcon</NxTable.Cell>
+                <NxTable.Cell>IconDefinition</NxTable.Cell>
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell>
+                  The support link has an icon to its left, by default it is a circle enclosing a question mark
+                  (<NxCode>faQuestionCircle</NxCode>) this specifies an alternate icon to be used.
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>supportText</NxTable.Cell>
+                <NxTable.Cell>string</NxTable.Cell>
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell>
+                  Alternate text that should appear in the support link, by default the text reads "Help and Support".
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>supportLink</NxTable.Cell>
+                <NxTable.Cell>string</NxTable.Cell>
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell>URL</NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>releaseText</NxTable.Cell>
```

Comment:
Should this be referred to as something with a bit more meaning? Like `productName`?

Reply:
> Should this be referred to as something with a bit more meaning? Like `productName`?

I avoided `productName` both because it's too similar to the text below (which I have since renamed `productName`) but also because it could take the form of `Release 1.1.1` or `Version 1.1.1`, etc

Reply:
> but also because it could take the form of Release 1.1.1 or Version 1.1.1, etc

I don't follow, wouldn't that be the `releaseNumber` not the `releaseText`?

Reply:
> > but also because it could take the form of Release 1.1.1 or Version 1.1.1, etc
> 
> I don't follow, wouldn't that be the `releaseNumber` not the `releaseText`?

In `Version 1.1.1` "version" is the `releaseText` and "1.1.1" is the `releaseNumber`.

Reply:
> In Version 1.1.1 "version" is the releaseText and "1.1.1" is the releaseNumber.

In that case, is "it could take the form of Release 1.1.1 or Version 1.1.1, etc" actually something we want to support? Or do we want to insist/enforce that it's always "<product name> x.y.z"?  The mockups just say "Product Release Number" on that line.

Reply:
> In that case, is "it could take the form of Release 1.1.1 or Version 1.1.1, etc" actually something we want to support? Or do we want to insist/enforce that it's always "<product name> x.y.z"? The mockups just say "Product Release Number" on that line.

A related question: the mockup shows this section as "Release 123" split across two lines when the sidebar is collapsed.  What is the intended relationship between that text and the text when it's expanded?

Reply:
> > In Version 1.1.1 "version" is the releaseText and "1.1.1" is the releaseNumber.
> 
> In that case, is "it could take the form of Release 1.1.1 or Version 1.1.1, etc" actually something we want to support? Or do we want to insist/enforce that it's always "<product name> x.y.z"? The mockups just say "Product Release Number" on that line.

There's actually one closed mockup that shows "Release 123" just to confuse matters. I am taking the position that we want to provide for "Text" +  "#" where both are optional and that the "Text" portion is _not_ visible in the collapsed state - which I agree does not match the mockups, but we can't know how long that text string is so it's either don't show it or truncate it. @dturner32 what is your guidance here?

Reply:
in the collapsed view, I say don't show the text in agreement with not knowing how long the text string could potentially be